ledger/src/blockstore.rs

```diff
@@ -1015,12 +1007,19 @@ impl Blockstore {
             || slot <= *last_root.read().unwrap())
     }
 
-    fn insert_coding_shred(
-        &self,
-        index_meta: &mut Index,
-        shred: &Shred,
-        write_batch: &mut WriteBatch,
-    ) -> Result<()> {
+    fn write_all(path: &str, payload: &[u8]) -> Result<()> {
+        let tmp_name = format!("{}.tmp", path);
+        let tmp_path = Path::new(&tmp_name);
+        let mut f = fs::File::create(tmp_path)?;
+        f.write_all(payload)?;
+        let real_path = Path::new(path);
+        //rename should sync force the metadata to be synced
```

Comment:
Oh I didn't know this, is this true (couldn't find any references in the docs)? A rename force syncs the file metadata?